Podfile.lock

```diff
@@ -511,7 +511,7 @@ DEPENDENCIES:
 SPEC REPOS:
   https://github.com/wordpress-mobile/cocoapods-specs.git:
     - WordPressAuthenticator
-    - WordPressKit
+    - WordPressUI
```

Comment:
I'm confused by the changes in these two pods 🤔 Why they jump from our internal spec to the public and vice versa? I tried to look at the merge commit that last touched this file but it's still unclear, I can see that WordPressKit went from a stable to a beta release, but I couldn't see changes to WordPressUI...

Reply:
@mokagio  See paaHJt-1SO-p2#what-changed-glitches as well as https://github.com/CocoaPods/CocoaPods/issues/10539